float64 left_wheel_speed
float64 right_wheel_speed

#like an enumerated type for the directions
uint8 DIRECTION_FORWARDS = 0
uint8 DIRECTION_BACKWARDS = 1

uint8 left_wheel_direction
uint8 right_wheel_direction
